_includes/reference/cloud-cli-template.md

```diff
@@ -0,0 +1,88 @@
+{% assign app = file.application %}
+
+{% assign commands = file.commands %}
+{% if app %}
+# {{ app.name }} (version {{ app.version }})
+{% else %}
+# Magento CLI
+{% endif %}
+
+This reference contains {{ commands | size }} commands available through the `magento/magento-cloud-cli` command-line tool.
+Generate the initial command list using `magento-cloud list` command at the {{ site.data.var.ece }} edition.
```

Comment:
This sentence is a little awkward.

Reply:
Oops! Yes. That template needs some help, actually.